src/poetry/packages/locker.py

```diff
@@ -61,6 +61,7 @@ def __init__(self, lock: str | Path, local_config: dict[str, Any]) -> None:
         self._local_config = local_config
         self._lock_data: TOMLDocument | None = None
         self._content_hash = self._get_content_hash()
+        self.contains_credential = False
```

Comment:
```suggestion
        self._contains_credential = False
```

Reply:
I changed every occurences of `_contains_credential` attribute.